components/Navbar/Navbar.styled.js

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+
+const StyledNavbar = styled.nav`
+  position: fixed;
+  bottom: 0;
+  background-color: var(--primary-color);
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 100vw;
+  height: 61px;
```

Comment:
This is a very specific value 😁 maybe 60px is still okay.
```suggestion
  height: 60px;
```